libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -9,16 +9,18 @@ import {
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
-import { Observable } from 'rxjs';
+import { debounceTime, distinctUntilChanged, filter, takeUntil } from 'rxjs/operators';
```

Comment:
Remove `filter` from here as it's unused now.
We should not have imports in all files we are touching.

Reply:
done